cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,37 @@
+class AddressStepPage {
+    private addressButton: string;
+    private accountPageURL: string;
+
+    constructor() {
+        this.addressButton = ".cart_navigation > .button > span";
+        this.accountPageURL = "http://automationpractice.com/index.php?controller=order";
+    }
+
+    public visitloginPageURL(): void {
+        cy.visit(this.accountPageURL)
+    }
+
+    public goTologinButton(): void {
+        cy.get(this.addressButton).click();
+    }
+}
+export { AddressStepPage}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Remove trailling lines. Last one should be line 19 (And should be empty line)

Reply:
already done in last commit